### ISSUE TITLE
Dimension Dialer Empty after update

### ISSUE THREAD

**Issue by xannor** [McJty/RFTools#2100]
Issue description: 
After updating my client to 1.16.4 from 1.16.3 if I try to open the dialing device I only get one entry in the list, which is the source transmitter. No receiver is listed, including the one that is about 7 block away. Also when I do this, ther are some NPE's in the log.

Steps to reproduce: 

Versions:
* Minecraft: 1.16.4
* Forge: 35.0.17
* CompatLayer (only if on Minecraft 1.10 or 1.11):
* McJtyLib: 5.0.13
* RFTools: (Base 2.0.6) (Utility 3.0.16)

Possibly incompatible mods and versions: 

Relevant logs, if any: 
https://pastebin.com/KNTULShs

This issue seems similar to #1937 but since it is for a different version I am not sure if it is the same problem.

**Comment by McJty** [comment]
When you did that update did you update other mods as well? Possibly remove mods?

**Comment by McJty** [comment]
Not sure what is wrong but I implemented a fix so that it shouldn't crash anymore

**Comment by xannor** [comment]
It was not a crash, just couldn't use the teleporters anymore. You fix does allow them to show up but it looks like it may be because a mod is not liking the move from .3 to .4 because the the receviers in the end and nether report invalid destination and my nether portal and end portal do not work. This is not your issue but I figured I would follow up and let you know the outcome, and that your fix did work as far as the recevires not displaying and the error that was logged. Thank you for the quick turn-around.

Also, as to the update/remove question, I did update the few that had updates, but I do not believe they had any dimensional interactions.
